tests/by-util/test_cksum.rs

```diff
@@ -1250,3 +1250,30 @@ fn test_several_files_error_mgmt() {
         .stderr_contains("empty: no properly ")
         .stderr_contains("incorrect: no properly ");
 }
+
+#[cfg(target_os = "linux")]
+#[test]
+fn test_non_utf8_filename() {
+    use std::ffi::OsString;
+    use std::os::unix::ffi::OsStringExt;
+
+    let scene = TestScenario::new(util_name!());
+    let at = &scene.fixtures;
+    let filename: OsString = OsStringExt::from_vec(b"funky\xffname".to_vec());
+
+    at.touch(filename.clone());
```

Comment:
A detail: instead of `clone()` you can simply use a reference. This also applies to lines 1268 and 1275.
```suggestion
    at.touch(&filename);
```